packages/driver/cypress/e2e/commands/actions/check.cy.js

```diff
@@ -323,24 +323,27 @@ describe('src/cy/commands/actions/check', () => {
     })
 
     it('can set options.waitForAnimations', () => {
-      cy.stub(cy, 'ensureElementIsNotAnimating').throws(new Error('animating!'))
```

Comment:
Because I moved `ensureElementIsNotAnimating` to `$actionability`, which is not public-facing, it's no longer trivial to stub. I've moved these tests to use an actual jQuery animation, and watch the count of command retries - this treats Cypress more as a 'black box', rather than peering inside at exactly how animation is determined.